code/platforms/win/edit.py

```diff
@@ -125,12 +125,9 @@ def select_none():
         #action(edit.select_paragraph):
         #action(edit.select_sentence):
     def select_word():
-        actions.key('ctrl-left ctrl-shift-right')
-        #action(edit.selected_text): -> str
-        #action(edit.sentence_end):
-        #action(edit.sentence_next):
-        #action(edit.sentence_previous):
-        #action(edit.sentence_start):
+        actions.edit.right()
```

Comment:
Quick flag that this strategy gets duplicated 3 times (once for each OS) and could probably be hoisted up one level of abstraction to the generic browser level avoid duplication